src/ac_copilot_trainer/modules/ws_bridge.lua

```
@@ -44,6 +58,118 @@ function M.configure(u)
   lastTry = -RECONNECT_SEC
   pendingCoaching = nil
   _recvQueue = {}
```

Luacheck (1.2.0)

[warning] setting non-standard global variable '_recvQueue'

(W111)